common/src/main/java/com/misterpemodder/shulkerboxtooltip/impl/renderer/BasePreviewRenderer.java

```diff
@@ -94,34 +95,34 @@ private void drawItem(ItemStack stack, int x, int y, TextRenderer textRenderer,
     x = this.slotXOffset + x + this.slotWidth * (slot % maxRowSize);
     y = this.slotYOffset + y + this.slotHeight * (slot / maxRowSize);
 
-    itemRenderer.renderInGuiWithOverrides(stack, x, y);
-    itemRenderer.renderGuiItemOverlay(textRenderer, stack, x, y, countLabel);
+    itemRenderer.renderInGuiWithOverrides(matrices, stack , x, y);
+    itemRenderer.renderGuiItemOverlay(matrices, textRenderer, stack, x, y, countLabel);
   }
 
-  protected void drawItems(int x, int y, int z, TextRenderer textRenderer,
+  protected void drawItems(MatrixStack matrices, int x, int y, TextRenderer textRenderer,
     ItemRenderer itemRenderer) {
-    float prevOffset = itemRenderer.zOffset;
+    // float prevOffset = itemRenderer.zOffset;
```

Comment:
You should avoid commit dead code. If variable not used anymore, simply delete it